build.gradle

```diff
@@ -1,10 +1,12 @@
 buildscript {
+    ext.kotlin_version = '1.0.7'
```

Comment:
🤔 shouldn't have to do this in the `android/build.gradle`...maybe clean and rebuild?
also we are behind a version of android studio right now which funks up syncing sometimes if you want to upgrade feel free to!